jqassistant.properties

```diff
@@ -1,15 +1,21 @@
 category=External Analyzers
 description=Analyze issues reported by jQAssistant.
 homepageUrl=https://github.com/jqassistant-contrib/sonar-jqassistant-plugin
-archivedVersions=1.9.0,1.10.0,1.11.0,1.12.0
-publicVersions=1.12.1
+archivedVersions=1.9.0,1.10.0,1.11.0,1.12.0,1.12.1
+publicVersions=2.0.0
 
 defaults.mavenGroupId=org.jqassistant.contrib.sonarqube
 defaults.mavenArtifactId=sonar-jqassistant-plugin
 
+2.0.0.description=Upgraded compatibility to jQAssistant 2.0 and SonarQube 9.9 (LTS)
+2.0.0.date=2023-08-27
+2.0.0.sqVersions=[9.9,LATEST]
+2.0.0.changelogUrl=https://github.com/jqassistant-tooling/sonar-jqassistant-plugin/blob/master/release-notes.adoc#200
+2.0.0.downloadUrl=https://search.maven.org/remotecontent?filepath=org/jqassistant/tooling/sonarqube/sonar-jqassistant-plugin/2.0.0/sonar-jqassistant-plugin-2.0.0.jar
+
 1.12.1.description=Improved sensor performance and ensure that all issues that cannot be attached to a file are displayed on project level
 1.12.1.date=2023-03-16
-1.12.1.sqVersions=[8.9,LATEST]
+1.12.1.sqVersions=[8.9,9.9.*]
```

Comment:
This range needs to end with the current version. Otherwise your users on 10.0-10.2 won't be prompted to upgrade. Since you're archiving this version, the overlap is fine.

Reply:
I updated to 10.2.*, hope this fits.

Reply:
Fine for me :)